ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Cleaner.java

```diff
@@ -201,16 +201,17 @@ private void clean(CompactionInfo ci, long minOpenTxnGLB) throws MetaException {
         LOG.debug("Cleaning based on writeIdList: " + validWriteIdList);
       }
 
+      final boolean[] removedFiles = new boolean[1];
```

Comment:
You can use org.apache.hive.common.util.Ref for this

Reply:
Done.

Reply:
+1